src/errors/test-run/templates.js

```diff
@@ -32,8 +32,8 @@ export default {
     `,
 
     [TEST_RUN_ERRORS.pageLoadError]: err => `
-        A request to ${formatUrl(err.url)} has failed.
-        Use quarantine mode to perform additional attempts to execute this test.
+        Failed to load a page at ${formatUrl(err.url)}.
+        Increase the "pageRequestTimeout", enable the "retryTestPages" option, or use quarantine mode to perform additional attempts to execute this test.
```

Comment:
```suggestion
        Failed to load the page at ${formatUrl(err.url)}.
        Future attempts to run this test may fail unless you increase the value of the "pageRequestTimeout" variable, enable the "retryTestPages" option, or enter quarantine mode.
```

Reply:
Future attempts may fail even if the user does what we tell them to.

```suggestion
        Failed to load a page at ${formatUrl(err.url)}.
        Increase the "pageRequestTimeout", enable the "retryTestPages" option, or enter quarantine mode.
```